### DISCUSSION TITLE
sign jwt token with JWKS

### DISCUSSION THREAD

**Discussion by vamsinm** [lestrrat-go/jwx#787] [Q&A]
i am new to golang. I am able to create jwt token and sign it with JWK using using jwt.Sign. i am trying to use JWKS to sign and not have any luck. 
```
privkey, err := jwk.ParseKey([]byte(jwkStr))
signed, err := jwt.Sign(tok, jwt.WithKey(jwa.RS256, privkey))
```
can anyone please give me example how to use JWKSet to signe JWT? Thank you!

**Comment by lestrrat** [comment]
Please show us what the exact problem is?
